### ISSUE TITLE
New Device Request: GoveeLife Mini Double Button Switch

### ISSUE THREAD

**Issue by chicknlil** [bwp91/homebridge-govee#845]
* **Which model is this? Need the `Hxxxx` number.**
H5126

* **Which type of device is this? Light, heater, leak sensor, etc.**
2 button switch

* **Which connection methods does this device have? Wifi? Bluetooth? LAN mode? All?**
Not entirely sure - it appers to ave connected itself to a distributted gateway, but Amazon listing says bluetooth.

* **Anything else?**
I was hoping to be able to pull this button through HB and use in HA.

**Comment by bwp91** [comment]
do you already own this device? are there homebridge logs to post?

**Comment by chicknlil** [comment]
I do own it, and it's visible in the Govee app, but Homebridge doesn't see it at all.

**Comment by bwp91** [comment]
i mean in the homebridge logs are there entries with data it says to post?

**Comment by chicknlil** [comment]
No, that's what I'm trying to say - Homebridge doesn't seem to be seeing it at all.

**Comment by bwp91** [comment]
okay understood! it’s just strange that no logs are popping up, as they should for any new model that is not implemented by the plugin.

i have to check: you have definitely added this device to the govee account for which you have the username and password in the plugin config?

**Comment by chicknlil** [comment]
![Screenshot_20240727-184634.png](https://github.com/user-attachments/assets/4a11e797-cb8c-4c2e-9641-f17342763693)

Indeed!

It never asked for my network credentials (though I can't recall if those are saved by the app), and it took a few attempts to get it connected via Bluetooth. 

It's got a button press history within the app - I can try being a button-presser (hah!) and see if that pops anything in the logs. 

Though really, I'll take any guidance you want to offer. 😁

**Comment by bwp91** [comment]
Please can you restart the plugin in homebridge and check the logs while the plugin is starting up, and just do one more check to see if any info about the device pops up?

**Comment by chicknlil** [comment]
[7/27/2024, 6:58:43 PM] [Govee] [Ben's Bedroom Button] is not currently supported, do create a Github issue with the following info:
{"device":"EE:25:C9:37:37:32:72:42","deviceName":"Ben's Bedroom Button","model":"H5126","httpInfo":{"groupId":0,"device":"EE:25:C9:37:37:32:72:42","sku":"H5126","spec":"","versionHard":"1","versionSoft":"1","deviceName":"Ben's Bedroom Button","deviceExt":{"deviceSettings":"{\"address\":\"C9:37:37:32:72:42\",\"bleName\":\"GV51267242\",\"ic\":0,\"sku\":\"H5126\",\"device\":\"EE:25:C9:37:37:32:72:42\",\"deviceName\":\"Ben's Bedroom Button\",\"versionHard\":\"1\",\"versionSoft\":\"1\",\"battery\":100,\"time\":1722120907430}","lastDeviceData":"{\"online\":true,\"bind\":true}","extResources":"{\"skuUrl\":\"https://d1f2504ijhdyjw.cloudfront.net/sku-img/4c945b48a243de461fe6572217878c10-add_list_type_device_5126.png\",\"headOnImg\":\"\",\"headOnImgNew\":\"\",\"headOffImg\":\"\",\"headOffImgNew\":\"\",\"ext\":\"\",\"ic\":0}"},"share":0,"deviceId":22673174,"gidConfirmed":false,"goodsType":145,"attributesId":0,"supportScene":0}}

**Comment by bwp91** [comment]
which platform are you running homebridge on? in the plugin initialisation logs do you see any warnings about the BLE client being disabled or enabled?

**Comment by chicknlil** [comment]
It's in a container on Proxmox, and yes, I believe I did see something to that effect.  Just restarted again and captured this:

[7/27/2024, 8:36:04 PM] [Govee] [BLE] disabling client as EAFNOSUPPORT, Address family not supported by protocol at new BluetoothHciSocketWrapped (/var/lib/homebridge/node_modules/homebridge-govee/node_modules/@abandonware/bluetooth-hci-socket/lib/native.js:9:5).

Looking at your Bluetooth Control section in your wiki (and it looks like I'll be bouncing to HB's section on Bluetooth shortly) ... before I do though, is there anything specific I should be doing "extra" in the plugin for it to see this? I saw in your example on the wiki, enabling BLE for lights, but this isn't a light so...

**Comment by chicknlil** [comment]
So I've been googling, and found your responses in [#236](https://github.com/bwp91/homebridge-govee/issues/236) and 

`hcitool dev` reveals no devices.  I'm also triple certain that bluetooth is up to date (on the proxmox node and HB's container). I've tried restarting HB a few times (after each 'fix' attempt) but still says no support. 

For what it's worth, HA sees 2 instances of Bluetooth (internal, external?):

![image](https://github.com/user-attachments/assets/d1424505-7911-44ca-b84f-4e9d0d71136c)

**Comment by chicknlil** [comment]
Just wanted to circle back, as much to update as to solicit suggestions.

The internal card is a Intel Corp. AX210, which is both BT and WiFi. There appears to be quite a few issues with this card and Proxmox currently in that both elements aren't working (I guess it's consistent at least). What's weird, is that as we can see by my HA screenshot, HA **is** seeing both BT devices (the internal one mentioned and a CSR-based dongle), but Proxmox claims it isn't, as I noted in that same previous reply. 

I found a discussion that suggested that HA and Proxmox are "fighting" for control of bluetooth (in general, not just this specific card), and when HA has it, Proxmox doesn't. Which...I guess I get. But I still need bluetooth outside of HA since I don't want to run HB in a container IN HA. Since I have two devices though, do I uninstall one from HA (the CSR-based dongle maybe?) and block Proxmox from using the internal?

Do you have any other ideas while I give this a go?
